.gitignore

```diff
@@ -75,3 +75,6 @@ fmod_editor.log
 /FMOD/**.cache
 /FMOD/**.restore
 game-jam-2.sln.DotSettings
+ProjectSettings/QualitySettings.asset
+ProjectSettings/Packages/com.unity.probuilder/Settings.json
```

Comment:
As much as I hate it, I wouldn't push this because if we ever actually update the settings, we'll want to make sure everyone gets them

Reply:
Gotcha, fixed